pilot/pkg/networking/core/v1alpha3/gateway_test.go

```diff
@@ -39,67 +37,10 @@ import (
 	"istio.io/istio/pilot/test/xdstest"
 	"istio.io/istio/pkg/config"
 	"istio.io/istio/pkg/config/schema/gvk"
-	tls_features "istio.io/istio/pkg/features"
 	"istio.io/istio/pkg/proto"
 )
 
 func TestBuildGatewayListenerTlsContext(t *testing.T) {
-	runTestBuildGatewayListenerTLSContext(t, &auth.TlsParameters{})
```

Comment:
why this commit was necessary?
I mean, this commit essentially makes this file equal to upstream. Does it mean we changed it before? Maybe in `fix test` commit? If so, it should be enough to remove it, right?

Reply:
The test was failing because [tlsParams](https://github.com/maistra/istio/pull/430/commits/a2dad5095041f24e3ee0fa4d095e29f664f4b9f0#diff-7abd8e9bc7226623690e85b5acc0280def2d72df7960b1324349b08dcb186ca1L458) was removed from upstream, which makes this option obsolete for most of the test. I tried to just pass empty params, but it wasn't used in the test. so lint throws warning.